aljuarismi/datasets.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2019 Shapelets.io
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import random as rng
+
+import click
+import pandas as pd
+
+num_rand = 0
```

Comment:
What is this rand number doing ?

Reply:
**It is for count the number of randoms we had made. When we create a random dataset, we will save it in the loaded_datasets.** Later on, if the user exit the program, it will ask if he want to save the actual workspace with the randoms we created, before the program close